app/scenes/Document/components/CommentThread.tsx

```diff
@@ -192,6 +214,19 @@ function CommentThread({
       onClick={handleClickThread}
     >
       {commentsInThread.map((comment, index) => {
+        if (collapse !== null) {
+          if (index === collapse.begin) {
+            const count = collapse.final - collapse.begin + 1;
+            return (
+              <ShowMore onClick={handleClickExpand} key="show-more">
+                {t("Show {{ count }} reply", { count })}
```

Comment:
Hm ... not show why it isn't pluralized. Doc says just use `count` as the variable, right?

Reply:
That's right, but the english translations need adjusting manually in this case.

Reply:
Mybad. I just discovered `build:i18n`. It works now :smile: